crowdsourcing/viewsets/task.py

```diff
@@ -129,6 +130,7 @@ def create(self, request, *args, **kwargs):
         serialized_data = {}
         if http_status == 200:
             serialized_data = TaskWorkerSerializer(instance=instance).data
+            update_worker_cache.delay([instance.worker_id], 'CREATED')
```

Comment:
Can you avoid using strings (like 'CREATED' here) at multiple places and define a type rather?